File: _CodeTopics/LeetCode_contest/weekly/weekly2021/231/231_2.py
class Solution(object):
    def minElements(self, nums, limit, goal):
        """
        :type nums: List[int]
        :type limit: int
        :type goal: int
        :rtype: int
        """
        
        diff = abs(goal - sum(nums))
        return diff/limit + int(bool(diff%limit))
    
"""
https://leetcode-cn.com/submissions/detail/151995578/

75 / 75 个通过测试用例
状态：通过
执行用时: 56 ms
内存消耗: 19.8 MB
"""

File: _CodeTopics/LeetCode/1601-1800/001785--weekly231_2/tran/231_2.cpp
class Solution {
public:
    int minElements(vector<int>& nums, int limit, int goal) {

        // 第 231 场周赛第二题
        // https://leetcode.cn/submissions/detail/151995578/

        long long diff = abs(goal - std::accumulate(nums.begin(), nums.end(), 0LL));
        return diff / limit + (diff % limit != 0? 1 : 0);
    }
};

/*
https://leetcode.cn/submissions/detail/389364694/

执行用时：
88 ms
, 在所有 C++ 提交中击败了
92.20%
的用户
内存消耗：
71.8 MB
, 在所有 C++ 提交中击败了
13.48%
的用户
通过测试用例：
77 / 77
*/
